sender/sms/sms_test.go

```diff
@@ -0,0 +1,33 @@
+package sms
```

Comment:
I figured you maybe hadn't written a unit test for Go before, so here's a real bare bones one (that's failing right now re: phone number formats).